src/main/java/org/influxdb/dto/Point.java

```diff
@@ -336,15 +326,18 @@ public String lineProtocol() {
      * @return the String without newLine
      */
     public String lineProtocol(final TimeUnit precision) {
-      final StringBuilder sb = CACHED_STRINGBUILDERS
-              .get()
-              .computeIfAbsent(this.measurement, MeasurementStringBuilder::new)
-              .resetForUse();
+      final StringBuilder sb = CACHED_STRINGBUILDERS.get();
```

Comment:
IMO declaring variables as final if they are (1) only visible inside a method and (2) not strictly required to be final only bloats the code.